README.md

```diff
@@ -83,6 +83,10 @@ npm test
 
 ## Change log
 
+### 0.6.6
```

Comment:
Let's add `(unreleased)` here, so we can remove it when we release, and we can append to the change log later.